pkg/ddl/ingest/backend.go

```diff
@@ -269,8 +269,14 @@ func (bc *litBackendCtx) ShouldSync(mode FlushMode) (shouldFlush bool, shouldImp
 	if mode == FlushModeForceLocalAndCheckDiskQuota {
 		shouldFlush = true
 	} else {
+		interval := bc.updateInterval
+		failpoint.Inject("mockSyncInterval", func(val failpoint.Value) {
```

Comment:
where is `mockSyncInterval`?

Reply:
Do you mean the way to enable `mockSyncInterval`?

First, we need to set the environment variable `GO_FAILPOINTS` to `github.com/pingcap/tidb/pkg/server/enableTestAPI=return` before startup, in order to turn on the ability to toggle failpoint dynamically through HTTP.

Second, send a PUT request to status port to change `mockSyncInterval`:

```
curl -X PUT http://127.0.0.1:10080/fail/github.com/pingcap/tidb/pkg/ddl/ingest/mockSyncInterval -d "return(3000)"
```